projects/canopy/src/lib/card/card-content/card-content.component.scss

```diff
@@ -0,0 +1,43 @@
+.lg-card-content--data-points {
+  display: grid;
+  grid-template: 'a a' 100% 'b c' 1fr;
+
+  .lg-data-point-label {
+    font-weight: var(--font-weight-bold);
+  }
+
+  .lg-data-point-value {
+    font-weight: var(--font-weight-light);
+  }
+
+  // first data point
```

Comment:
I'm just being annoying here, could the first letter be capital please? `First data point`

Reply:
done 👍